storage/hw_raid/main.sh

```diff
@@ -52,29 +52,7 @@ kdump_prepare()
 
 #Function to install FIO
 function install_fio() {
-
-    rlRun "which fio"
-    if [ $? -eq 0 ]; then
-        rlLog "INFO: fio already installed"
-        return
-    fi
-    rlRun "$YUM -y install fio"
-    if [ $? -eq 0 ]; then
-        rlLog "INFO: fio succesfully installed by yum -y install fio"
-        return
-    fi
-
-    git_url=git://git.kernel.org/pub/scm/linux/kernel/git/axboe/fio.git
-    rlRun "$YUM install libaio-devel zlib-devel gcc -y"
-    rlRun "git clone $git_url"
-    rlLog "INFO: Installing Fio"
-    rlRun "cd fio && ./configure && make && make install"
-    rlRun "which fio"
-    if [ $? -ne 0 ]; then
-        rlLog "FAIL: Fio install failed"
-        return 1
-    fi
-    rlLog "INFO: Fio succesfully installed"
+    rlRun "rpm -q fio || $YUM -y install fio"
```

Comment:
So we're missing the capacity to install from Git? Is that ok/on purpose?

Reply:
Thanks Esteban. I removed ability to install FIO from Git on purpose as it is not necessary at this time. Additionally, I moved install_fio as you noticed as it only needs to install once.

Reply:
Cool, thanks for confirming. Could you add that information to the commit message, so it's available to anyone maintaining that code afterwards? Also it would be nice to have a more descriptive commit summary 😄